x-pack/plugins/security_solution/server/ui_settings.ts

```diff
@@ -112,6 +116,31 @@ export const initUiSettings = (uiSettings: CoreSetup['uiSettings']) => {
       requiresPageReload: true,
       schema: schema.boolean(),
     },
+    [DEFAULT_RULES_TABLE_REFRESH_SETTING]: {
+      name: i18n.translate('xpack.securitySolution.uiSettings.rulesTableRefresh', {
+        defaultMessage: 'Rules auto refresh',
+      }),
+      description: i18n.translate(
+        'xpack.securitySolution.uiSettings.rulesTableRefreshDescription',
+        {
+          defaultMessage:
+            '<p>Enables auto refresh on the all rules and monitoring tables, in milliseconds</p>',
+        }
+      ),
+      type: 'json',
+      value: `{
+  "on": ${DEFAULT_RULE_REFRESH_INTERVAL_ON},
+  "value": ${DEFAULT_RULE_REFRESH_INTERVAL_VALUE},
+  "idleTimeout": ${DEFAULT_RULE_REFRESH_IDLE_VALUE}
+}`,
+      category: ['securitySolution'],
```

Comment:
Wondering if this whole file should have used `APP_ID` from constants at some point instead of the hard coded names of `securitySolution`. No changes if you don't want to, but I would try to update that and see if it works? I am curious if has to be this hard coded name and cannot use the variable we have.

Reply:
I went ahead and changed it to use `APP_ID` in https://github.com/elastic/kibana/pull/83023 - it seemed to work just fine on testing it.